src/Storage/Repository/PgInstanceRepository.cs

```diff
@@ -295,7 +295,7 @@ await using (NpgsqlDataReader reader = await pgcom.ExecuteReaderAsync())
         }
 
         /// <inheritdoc/>
-        public async Task<(Instance Instance, long InternalId)> GetOne(int instanceOwnerPartyId, Guid instanceGuid, bool includeElements = true)
+        public async Task<(Instance Instance, long InternalId)> GetOne(Guid instanceGuid, bool includeElements = true)
```

Comment:
I think we should change localtest to use the normal postgres implementation rather than a local mock with flat files. Until we do that we can require that the guid is unique across partyid and keep the existing local hierarchy and search in the subdirectories.

Reply:
Okay, I gues there exists alternatives still.